lib/tasks/coverage.js

```diff
@@ -15,57 +13,55 @@ const log = require('fancy-log');
 
 const configure = function configure (gulp, opts, env) {
   let npmBin;
-  gulp.task('npm-bin', function () {
+  gulp.task('npm-bin', async function () {
     if (npmBin) {
       return B.resolve();
     }
-    return exec('npm bin').then(function (bin) {
-      if (Array.isArray(bin)) {
-        bin = bin[0];
-      }
-      npmBin = bin.trim();
-      log(`Determined npm bin: ${npmBin}`);
-    });
+    let bin = await exec('npm bin');
```

Comment:
`const [stdout, stderr] = ...` format might be easier to read

Reply:
This is `child_process#exec`. There is no form like that.